build.rs

```diff
@@ -1,17 +1,21 @@
+use std::env;
 use std::process::Command;
 fn main() {
     println!("cargo:rerun-if-changed=Dockerfile.nym");
-    let output = Command::new("docker")
-        .arg("build")
-        .arg("-t")
-        .arg("nym:latest")
-        .arg("-f")
-        .arg("./Dockerfile.nym")
-        .arg(".")
-        .output()
-        .expect("ERROR: unable to prepare context");
-    println!("status: {}", output.status);
-    println!("stdout: {}", String::from_utf8_lossy(&output.stdout));
-    println!("stderr: {}", String::from_utf8_lossy(&output.stderr));
-    assert!(output.status.success());
+    let docker_build = env::var("DOCKER_BUILD").is_ok();
+    if docker_build {
```

Comment:
```suggestion
    if env::var("DOCKER_BUILD").is_ok() {
```
bit of a nit :p

Reply:
hahaha I actually prefer having a separate variable for verbosity. So if we ever need to debug this, all we need to do is:
`dbg!(docker_build);` :P